curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -296,8 +296,10 @@ int FileCacheManager::Read(uint64_t inodeId, uint64_t offset, uint64_t length,
     }
 
     {
-        unsigned int maxRetry = 3;  // hardcode, fixme
-        unsigned int retry = 0;
+        uint32_t maxRetry =
+            s3ClientAdaptor_->GetMaxReadRetry();  // hardcode, fixme
```

Comment:
The default here should be infinite retries. It is recommended to use 0 to indicate infinite retries

Reply:
maxRetry is changed to maxIntervalNum, which means the maximum number of intervals